src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/5.diff.sql

```diff
@@ -0,0 +1,1613 @@
+/****************************************************************************************
+Guidelines to create migration scripts - https://github.com/microsoft/healthcare-shared-components/tree/master/src/Microsoft.Health.SqlServer/SqlSchemaScriptsGuidelines.md
+******************************************************************************************/
+SET XACT_ABORT ON
+
+BEGIN TRANSACTION
+
+---BEGIN - CREATE/ALTER Statements----------
+
+IF NOT EXISTS (
+    SELECT *
+    FROM sys.tables
+    WHERE name = 'Partition')
+BEGIN
+    CREATE TABLE dbo.Partition (
+        PartitionKey            BIGINT                 NOT NULL, --PK
+        PartitionId             VARCHAR(64)            NOT NULL    DEFAULT 'Microsoft.Default',
```

Comment:
Should we control the default from Compute. Here it does not fully understand we to default and when to error

Reply:
Also see if another name is less confusing

Reply:
When you say another naming, you mean to change `Microsoft.Default` ?

Reply:
Also thinking twice we dont need a default here since we are already handling by inserting Microsoft.Default always.

Reply:
Moving default values to compute and changing the column name to `PartitionName` to make the usage clearer.